.github/workflows/ci.yml

```
@@ -64,7 +64,7 @@ jobs:
           lichen --config=../../.github/license.yaml action
 
   test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-24.04 
```

Nit I'm a decent fan of pinning versions whenever possible in CI unless there is documentation/reasoning behind specifying latest. Is there a good reason why we should be using latest that I'm not privy to?

---

Yeah good call out. TLDR: was fighting the ubuntu-20.04 deprecation and went with latest. I think this is one of the few times its okay as long as we do not depend on tooling specific to ubuntu-24.04.